src/components/Header/index.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import { useState } from 'react';
+import { NavLink } from "react-router-dom";
+import { logout } from "../../services/Authentication";
+
+function CompanyName({ name, classes }) {
+    return (
+        <div>
+            <img className="h-16" src={require('../../logoRefer.png')} alt="logo" />
+        </div>
+    )
+}
+
+function MenuLink({ text, route }) {
+
+    const classes = {
+        active: "text-xl bg-white text-blue-500 bg-opacity-90 my-2 px-2 py-1 rounded-md",
+        inActive: "text-xl text-white bg-opacity-90 my-2 px-2 py-1 rounded-md transform hover:bg-white hover:bg-opacity-90 hover:text-blue-500"
```

Comment:
This is the normal state of the link, so the class can be called like that
```suggestion
        link: "text-xl text-white bg-opacity-90 my-2 px-2 py-1 rounded-md transform hover:bg-white hover:bg-opacity-90 hover:text-blue-500"
```

Reply:
Changed.